packages/vscode-extension/src/project/project.ts

```diff
@@ -236,8 +238,10 @@ export class Project
     }
   }
 
-  public async goHome() {
-    await this.reloadMetro();
+  public async goHome(homeUrl: string) {
+    if (this.expoRouterInstalled) {
+      await this.openNavigation(homeUrl);
+    }
```

Comment:
What if it's not installed?

Reply:
It that case we reload metro. I've changed it.